packages/fiori3/src/components/AnalyticalTable/pagination/index.tsx

```diff
@@ -0,0 +1,213 @@
+import React, { Component, FC, Fragment } from 'react';
+import { ClassProps } from '../../../interfaces/ClassProps';
+import { Label } from '../../../webComponents/Label';
+import { Link } from '../../../webComponents/Link';
+import { ContentDensity } from '../../../enums/ContentDensity';
+import { JSSTheme } from '../../../interfaces/JSSTheme';
+import { withStyles } from '@fiori-for-react/styles';
+import { LinkType } from '@fiori-for-react/fiori3';
+
+enum NavigationModes {
+  rightArrowPress,
+  leftArrowPress,
+  default
+}
+
+interface LinkHOCProps {
+  enabled?: boolean;
+  page?: number;
+  selectedPage: number;
+  children: string | number;
+  mode?: NavigationModes;
+  onClick: (page: number, mode: NavigationModes) => void;
+}
+
+const onClickHandler = (props) => (e) => {
+  const { page, selectedPage, onClick, mode } = props;
+
+  if (page === selectedPage) {
+    return;
+  }
+  if (page > -1) {
+    onClick(page, NavigationModes.default);
+  } else {
+    onClick(selectedPage, mode);
+  }
+};
+
+export const LinkHOC: FC<LinkHOCProps> = (props) => {
+  const { children, page, selectedPage } = props;
+
+  return (
+    <Link
+      style={{ paddingRight: '0.2rem' }}
+      onPress={onClickHandler(props)}
+      type={page === selectedPage ? LinkType.Emphasized : LinkType.Default}
+    >
+      {children.toString()}
+    </Link>
+  );
+};
+
+LinkHOC.defaultProps = {
+  children: '',
+  enabled: true,
+  page: -1
+};
+
+const styles = ({ theme, parameters, contentDensity }: JSSTheme) => ({
+  container: {
+    padding: '16px 20px',
+    height: `${contentDensity === ContentDensity.Compact ? '2rem' : '3rem'}`,
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: parameters.sapUiListHeaderBackground,
+    '& a': {
```

Comment:
does this selector still apply? I would guess it's now `ui5-link` because it's the web component

Reply:
we discussed this and i changed it to HSLColor.of(sapUiListBackground).darken(5).hsl; 
i don't really get it.